src/pages/home.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect, useState } from "react";
+import { wrapper } from "@/styles/spacing";
+import { hackerApplications } from "@/interfaces/fb-int";
+import { Timestamp } from "firebase/firestore";
+
+const Home = () => {
+    const [tableData, setTableData] = useState<Map<String, hackerApplications>>(
+        new Map()
+    );
+
+    const hacker1: hackerApplications = {
+        applicantId: "IDK MAN?",
+        applicationDate: Timestamp.now(),
+        hackathonTerm: "F23",
+        notifyByEmail: true,
+        notifyBySMS: false,
+        inAcceptedList: true,
+        finalDecision: "accepted",
+        type: "hacker",
+    };
+    const hacker2: hackerApplications = {
+        applicantId: "IDK MAN123?",
+        applicationDate: Timestamp.now(),
+        hackathonTerm: "W23",
+        notifyByEmail: true,
+        notifyBySMS: false,
+        inAcceptedList: true,
+        finalDecision: "rejected",
+        type: "hacker",
+    };
+
+    useEffect(() => {
+        const map: Map<String, hackerApplications> = new Map();
+        map.set(hacker1.applicantId, hacker1);
+        map.set(hacker2.applicantId, hacker2);
+        setTableData(map);
+        console.log(map);
+    }, []);
+
+    // Styles
+    const tableBorder: React.CSSProperties = {
+        border: "1px solid black",
+    };
+
+    const tableColumns: React.CSSProperties = {
+        backgroundColor: "aqua",
+        paddingLeft: "20px",
+        paddingRight: "20px",
+        border: "1px solid black",
+    };
+
+    const tableHeaderColumns: React.CSSProperties = {
+        backgroundColor: "royalblue",
+        paddingLeft: "20px",
+        paddingRight: "20px",
+        border: "1px solid black",
+    };
+
```

Comment:
Could these styles be done using tailwindcss instead? If not, we can leave it here for now, as soon as everyone is done with the table we can refactor.

I just want to keep the code base as consistent as possible.